iOS/Chameleon/source/Chameleon/ApiDefinition.cs

```diff
@@ -16,524 +16,591 @@
-		void SetStatusBarStyle (StatusBarStyle statusBarStyle);
-	}
-
-	// @interface Chameleon (UIButton)
-	[Category]
-	[BaseType (typeof(UIButton))]
-	interface UIButtonExtensions
-	{
-		// -(void)setSubstituteFontName:(NSString *)name __attribute__((annotate("ui_appearance_selector")));
-		[Export ("setSubstituteFontName:")]
-		void SetFont (string name);
-	}
-
-	// @interface Chameleon (UILabel)
-	[Category]
-	[BaseType (typeof(UILabel))]
-	interface UILabelExtensions
-	{
-		// -(void)setSubstituteFontName:(NSString *)name __attribute__((annotate("ui_appearance_selector")));
-		[Export ("setSubstituteFontName:")]
-		void SetFont (string name);
-	}
-
-//	// @interface ChameleonPrivate (UIColor)
-//	[Category]
-//	[BaseType (typeof(UIColor))]
-//	interface UIColor_ChameleonPrivate
-//	{
-//		// @property (readwrite, nonatomic) NSUInteger count;
-//		[Export ("count")]
-//		nuint GetCount ();
-//	
-//		// @property (readwrite, nonatomic) NSUInteger count;
-//		[Export ("setCount:")]
-//		void SetCount (nuint value);
-//	
-//		// +(UIColor *)colorFromImage:(UIImage *)image atPoint:(CGPoint)point;
-//		[Static]
-//		[Export ("colorFromImage:atPoint:")]
-//		UIColor ColorFromImage (UIImage image, CGPoint point);
-//
-//		// -(UIColor *)colorWithMinimumSaturation:(CGFloat)saturation;
-//		[Export ("colorWithMinimumSaturation:")]
-//		UIColor ColorWithMinimumSaturation (nfloat saturation);
-//
-//		// -(BOOL)isDistinct:(UIColor *)color;
-//		[Export ("isDistinct:")]
-//		bool IsDistinct (UIColor color);
-//
-//		// -(BOOL)getValueForX:(CGFloat *)X valueForY:(CGFloat *)Y valueForZ:(CGFloat *)Z alpha:(CGFloat *)alpha;
-//		[Export ("getValueForX:valueForY:valueForZ:alpha:")]
-//		bool GetValueForX (out nfloat X, out nfloat Y, out nfloat Z, out nfloat alpha);
-//
-//		// -(BOOL)getLightness:(CGFloat *)L valueForA:(CGFloat *)A valueForB:(CGFloat *)B alpha:(CGFloat *)alpha;
-//		[Export ("getLightness:valueForA:valueForB:alpha:")]
-//		bool GetLightness (out nfloat L, out nfloat A, out nfloat B, out nfloat alpha);
-//	}
-
-//	// @interface ChameleonPrivate (UIImage)
-//	[Category]
-//	[BaseType (typeof(UIImage))]
-//	interface UIImage_ChameleonPrivate
-//	{
-//		// +(UIImage *)imageWithImage:(UIImage *)image scaledToSize:(CGSize)newSize;
-//		[Static]
-//		[Export ("imageWithImage:scaledToSize:")]
-//		UIImage ImageWithImage (UIImage image, CGSize newSize);
-//
-//		// -(UIImage *)imageScaledToSize:(CGSize)newSize;
-//		[Export ("imageScaledToSize:")]
-//		UIImage ImageScaledToSize (CGSize newSize);
-//	}
-
-//	// @interface ChameleonPrivate (UIView)
-//	[Category]
-//	[BaseType (typeof(UIView))]
-//	interface UIView_ChameleonPrivate
-//	{
-//		// -(BOOL)isTopViewInWindow;
-//		[Export ("isTopViewInWindow")]
-//		bool IsTopViewInWindow ();
-//
-//		// -(UIView *)findTopMostViewForPoint:(CGPoint)point;
-//		[Export ("findTopMostViewForPoint:")]
-//		UIView FindTopMostViewForPoint (CGPoint point);
-//	}
+    [Static]
+    partial interface Constants
```

Comment:
I recommend moving these constants to Chameleon class to avoid breaking changes. If breaking changes are pretended, also, I recommend to move them to the main class.